### PR TITLE
RAC-5031 - Conversion and testing of CIT Skupack tests

### PR DESCRIPTION
**Pull request by larry-dean**
* 	Reworked skupack_tests.py to better handle post/put/patch.
* 	Created skupack_node_updates.py to test sku attachement and de-attachment
* 	flake8 corrections

### CODE REVIEW

**Comment by larry-dean** [comment]
@jimturnquist @hohene @johren

**Comment by hohene** [comment]
test this please

**Comment by hohene** [comment]
test this please

File: test/tests/api/fit-staging/v2_0/skupack_tests.py [hunk 1]
```diff
@@ -1,134 +1,160 @@
...
+from on_http_api2_0.rest import ApiException
+from json import loads, dumps, dump
+from nosedep import depends
+from nose.plugins.attrib import attr
 
+logs = flogging.get_loggers()
 
-LOG = Log(__name__)
 
+# @test(groups=['skus_api2.tests'])
```
Comment (johren) [inline test/tests/api/fit-staging/v2_0/skupack_tests.py]: Is this line still needed?

File: test/tests/api/fit-staging/v2_0/skupack_node_updates.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+            for item in data:
+                if item["id"] == node_id:
+                    flag = True
+                    break
+            self.assertTrue(flag, msg='Node id {} is not assocaited the original sku id {}'.format(node_id, original_sku_id))
+
+    def find_node_with_sku(self, node_list):
+        '''
+            Find a compute node with an assocaited sku attached
+            If not node is found with an attached sku, None is
```
Comment (johren) [inline test/tests/api/fit-staging/v2_0/skupack_node_updates.py]: Typo:  If no node

File: test/tests/api/fit-staging/v2_0/skupack_tests.py [hunk 3]
```diff
@@ -139,200 +165,256 @@ def get_sku_nodes(self):
...
+            logs.info("POST Failed. status: %s, Message: %s", e.status, e.message)
+            return
+
+        # sku pack posted
+        logs.info("****** Posted sku pack: %s", res.text)
+        self.assertIsNotNone(res, msg='POST: Connection could not be established')
+        self.assertEqual(201, res.status_code, msg="expected 201, received {0}".format(res.status_code))
+        self.__packFolderId = res.text.split('"')[3]
+
+        logs.info(" FoldererId: %s ", self.__packFolderId)
```
Comment (johren) [inline test/tests/api/fit-staging/v2_0/skupack_tests.py]: Folderer?

**Comment by hohene** [comment]
test this please

**Pull request merged**